processor/processor.go

```diff
@@ -1750,8 +1757,8 @@ func (proc *Handle) preprocessStage(partition string, subJobs subJob) (*preTrans
 			continue
 		}
 
-		for _, enricher := range proc.enrichers {
-			if err := enricher.Enrich(source, &gatewayBatchEvent); err != nil {
+		for _, e := range proc.enrichers {
+			if err := e.Enrich(source, &gatewayBatchEvent); err != nil {
```

Comment:
why do we prefer singler letter variables?

Reply:
Conflict with package name and single letter is in a single line loop so it should be fine :+1: